src/client/datascience/jupyter/jupyterSession.ts

```diff
@@ -34,6 +34,8 @@ import { JupyterKernelPromiseFailedError } from './jupyterKernelPromiseFailedErr
 import { JupyterWaitForIdleError } from './jupyterWaitForIdleError';
 import { createJupyterWebSocket } from './jupyterWebSocket';
 
+export const internalUseCellKey: string = "#%DATASCIENCE_INTERNAL_KEY%#";

```

Comment:
Looks to me like this should live in one of the types files. It's not really relevant here. Maybe types.ts at the datascience level?

Reply:
Yeah, I didn't know what the best place for this was. If you think types.ts is it, then I'll change it to that.